src/debug/tester/Kconfig

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config COMP_TESTER
+	tristate "tester module"
+	default n
+	depends on IPC_MAJOR_4
+	help
+	  Build a tester module. To be used in continous 
+	  integration process for special test cases that
+	  require a test code injected into the system itself
```

Comment:
This is better description , thanks. It's still complex way to express this. I.i.e. without context, "test code injected into the system itself" sounds like this module will try to modify system firmware or some other black magic. But I get your point, you want to run test code in an audio module. But yeah, maybe this is sufficient now, but I'd still think a simpler "audio test module that can create synthetic load" would be easier to understand.

The git commit description I guess is unmodified. That's less important than this Kconfig doc, but room to improve understandability there as well.